src/pages/LoginPage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import Header from '../components/Header_type_1';
+import Footer from '../components/Footer';
+import Login2FA from '../components/Login2FA';
+import LoginWeb3 from '../components/LoginWeb3';
+import LoginNeuKey from '../components/LoginNeuKey';
+
+import styles from './LoginPage.scss';
+import common from '../styles/common.scss';
+
+export default () =>
+  (<div className={common.appContainer}>
+    <Header />
+    <div className={common.contentArea}>
+      <div className={common.widthLimiter}>
+        <Grid fluid>
+          <Row>
+            <Col lgOffset={1} lg={10}>
+              <Row>
```

Comment:
If we do 1 column offset then 3 x 3 columns we will end with 10 columns. So on left side there will be one column and two on right and content it won't be centred.
Nesting grids is feature of flebox layout described in doc. Maybe (didn't try it) mixing fixed size columns and auto width feature would work. Something like
```
<Col lgOffset={1} lg>
<Col lg>
<Col lg>
<Col lg={1}>
```
but then we would end with extra empty column.